nautobot_netbox_importer/generator/fields.py

```diff
@@ -23,24 +37,12 @@ def role(adapter: SourceAdapter, source_content_type: str) -> SourceFieldDefinit
         role_wrapper = adapter.configure_model(
             source_content_type,
             nautobot_content_type="extras.role",
-            fields={"color": "color", "content_types": "content_types"},
```

Comment:
why not necessary content_types?

Reply:
Described above, `content_types` fields are auto added.